support/root/jest-playwright.config.cjs

```diff
@@ -2,14 +2,15 @@
 
 const { E2E_DEBUG, E2E_COVERAGE, E2E_BROWSER = 'chromium' } = process.env;
 
-const browsers = E2E_BROWSER.split(',');
+/** @type {?} */
+const browsers = E2E_BROWSER.split(',')
```

Comment:
This is equivalent to `const browsers: unknow = E2E_BROWSER.split(',')`. 

If I don't add this comment, `import('jest-playwright-preset').JestPlaywrightConfig` will compain that `browsers` should be `Array<"firefox" | "chromium" | "webkit">` instead of `string[]`.

I want to write something like this but JSDoc/TS doesn't allow me to do it.

```
/* @type {Array<"firefox" | "chromium" | "webkit">} */
const browsers = E2E_BROWSER.split(',')
```